service/src/config.rs

```diff
@@ -185,10 +185,9 @@ pub struct NetworkSubgraph {
         long,
         value_name = "network-subgraph-endpoint",
         env = "NETWORK_SUBGRAPH_ENDPOINT",
-        default_value_t = String::from("https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-goerli"),
         help = "Endpoint to query the network subgraph from"
     )]
-    pub network_subgraph_endpoint: String,
+    pub network_subgraph_endpoint: Option<String>,
```

Comment:
Why making it optional? I'd think that the indexer-service cannot operate without being able to query the network subgraph.

Reply:
ah yes you are right, changing it back!